packages/cashier/src/components/cashier-container/real/real.tsx

```diff
@@ -6,17 +6,12 @@ import { useCashierStore } from '../../../stores/useCashierStores';
 import './real.scss';
 
 const Real = observer(({ is_deposit = false }: { is_deposit?: boolean }) => {
-    const {
-        traders_hub: { is_low_risk_cr_eu_real },
-        ui: { is_dark_mode_on },
-    } = useStore();
-
```

Comment:
it's not that important but just for having more consistency inside our code base I'm asking this. I saw in [this](https://github.com/binary-com/deriv-app/pull/8707#pullrequestreview-1440525690) PR we are getting our data from store like this: 
```
const {
        common: { current_language },
    } = useStore();
```
but here you actually changed the syntax. IMO the way you wrote it is more clear in the first look , So I think we can discuss and agree on one of the formats  for the future

Reply:
@shayan-deriv Yeah agree with you, I personally try to avoid nested object destructuring cause it will be hard to follow by eye. It similar to nested ternary operator situation 👀